modules/ROOT/pages/to-request-access.adoc

```diff
@@ -4,19 +4,19 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-To register a client application to an existing API, the client application
-must first request access. When the request is approved by the API owner, a
-contract is created between the client application and the API, and the client
+To register a client application to an existing API or API Group asset, the client application
+must first request access. When the request is approved by the asset owner, a
+contract is created between the client application and the asset, and the client
 application is registered.
 
-APIs that are protected by a client ID enforcement policy require client
+Assets that are protected by a client ID enforcement policy require client
```

Comment:
"Instances (be it for APIs or API Groups)"

Reply:
Fixed.